chainer/training/triggers/manual_schedule_trigger.py

```diff
@@ -18,12 +18,18 @@ class ManualScheduleTrigger(object):
         unit (str): Unit of the time specified by ``points``. It must be
             either ``'iteration'`` or ``'epoch'``.
 
+    Attributes:
+        finished (bool): Flag that triggered when this trigger will never
+            called again. The flag helps decision to call
+            `Extension.initialize` or not in `trainer`.
```

Comment:
Suggestion to improve:
"Flag that indicates whether or not this trigger will fire in the future. This flag is used to determine if the extension should be initialized after resume."

Reply:
Sorry for the late response, and thank you for your suggestion.
I reflected it to the latest version.